save-agent/src/linuxX64Main/resources/agent.properties

```diff
@@ -5,4 +5,5 @@ heartbeat.intervalMillis=15000
 requestTimeoutMillis=1000
 executionDataRetryAttempts=5
 executionDataInitialRetryMillis=1000
-debug=true
\ No newline at end of file
+debug=true
+cliCommand=echo Provide CLI command here
```

Comment:
It is just stub line, but why with `echo`?

Reply:
No actual reason; just so that it looks more like a command

Reply:
Ok